### DISCUSSION TITLE
Appropriate licence for Opera GX browser

### DISCUSSION THREAD

**Discussion by pete0000001** [aclap-dev/video-downloadhelper#1562] [Help and Q&A]
### Your question

Hi, I have been using VDH extension with Opera GX successfully and I wish to get a premium licence now.  I have clicked on the 'get a conversion licence' button which has sent me to https://www.downloadhelper.net/convert - to purchase one. The licence shows as "Lifetime License for Firefox" is this correct for Opera GX browser ?

### Addon & CoApp version and details

Version 9.1.1.0 (stable), target google, locale en-US.

**Comment by Mattmjs** [comment]
Which extension are you using ? I believe Opera GX is chromium based meaning you should have the chrome extension.
It could be the Chrome license your after but that doesn't explain why it's offering the Firefox version. What does Opera GX user agent
look like if you can share that here.

**Comment by jcfrog** [comment]
Hi,
Yes, probably a user agent issue: we do not officially support Opera GX. If you need a refund to purchase Chrome premium, please send us an email at licenses@downloadhelper.net
The links to purchase the correct premium: https://github.com/aclap-dev/video-downloadhelper/wiki#go-premium

**Comment by Cronosus** [comment]
chrome licence works in opera gx
